pkg/component/worker/kernelsetup_linux.go

```diff
@@ -83,3 +83,12 @@ func KernelSetup() {
 	enableSysCtl("net/bridge/bridge-nf-call-iptables")
 	enableSysCtl("net/bridge/bridge-nf-call-ip6tables")
 }
+
+// KernelMajorVersion returns the major version number of the running kernel
+func KernelMajorVersion() byte {
```

Comment:
At some point, we should consolidate that with the code of the sysprobes.

Reply:
I was not aware we had `osRelease` there. Is there any convenient way to get kernel major version (or just `osrelease` string) using sysprobes?

Reply:
No, not really ATM. This was more like a "note to self" / "raise awareness" comment that we should probably make the sysprobes stuff a bit more usable outside of the startup probe context, as we start to rely more and more on those things in core k0s code. This is definitely something to be addressed separately.

Reply:
For reference: https://github.com/k0sproject/k0s/blob/v1.25.2%2Bk0s.0/internal/pkg/sysinfo/probes/linux/kernel.go#L37-L52

Totally unusable outside of a sysprobe, currently.

Reply:
ok. maybe we just leave it as it is for now then.

Reply:
Sure!